### ISSUE TITLE
[Feature Req] Daemon mode

### ISSUE THREAD

**Issue by caoli5288** [sorz/sstp-server#20]
A daemon mode which like ss's is good. `sstpd -f /etc/sstpd.ini -l /var/log/sstpd.log -d start` make daemon start and redirect output to `/var/log/sstpd.log`.

`nohup` is evil.

**Comment by sorz** [comment]
See #12, and [sstp-server@.service](https://github.com/sorz/archlinux-packages/blob/master/sstp-server/sstp-server%40.service) may help.

**Comment by alex-eri** [comment]
Maybe move it here?

**Comment by caoli5288** [comment]
Yes. `supervisor` is good. but it still some issues in el6(python-2.6 and gcc-4.4) that we can't install `sstp-server` and manage it by `supervisor` directly. I'll write blog to guide how to fix it.

**Comment by alex-eri** [comment]
i use cython or/and cxfreeze and make executables to port my apps to legacy distributions.
